examples/src/keyrings/hierarchical_keyring.py

```diff
@@ -0,0 +1,204 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Example showing basic encryption and decryption of a value already in memory."""
+import sys
+
+import boto3
+# ignore missing MPL for pylint, but the MPL is required for this example
+# noqa pylint: disable=import-error
+from aws_cryptographic_materialproviders.keystore.client import KeyStore
```

Comment:
Question: Is it possible we could reference `KeyStore` in  `aws_cryptographic_materialproviders.keystore.__init__`,
such that users could write
`from aws_cryptographic_materialproviders.keystore import KeyStore`
?

Reply:
That should possible, but it certainly crosses into the territory of questionable namespace hacking...

Reply:
I think it's reasonable for the primary local service client class to be accessible from this module. Will update.